terraform/global/um-pep-engine/config/pep-engine.tf

```diff
@@ -76,14 +76,19 @@ resource "kubernetes_service" "pep-engine" {
   }
 }
 
+
 resource "kubernetes_deployment" "pep-engine" {
   metadata {
     name   = "pep-engine"
     labels = { app = "pep-engine" }
   }
+<<<<<<< HEAD
+  
+=======
 
   depends_on = [null_resource.waitfor-login-service]
 
+>>>>>>> develop
```

Comment:
Be careful with this merge added content such as:
<<<< HEAD
which will break the terraform files (cannot be parsed).

Clean up this files, use develop branch as authoritative